configuration/authentication.mdx

```diff
@@ -366,6 +366,37 @@ Once enabled, client tokens can be retrieved by sending a Kubernetes pod's servi
 
 Further explanation for using this method can be found in the [Authentication: Kubernetes](/authentication/methods#kubernetes) documentation.
 
+#### Troubleshooting
+
+**verifying service account: failed to verify signature: fetching keys oidc**
+
+In some managed Kubernetes cluster environments, the default cluster OIDC provider is replaced with the platforms managed alternative.
+For example, EKS clusters leverage this so that they can issue service accounts tokens which can assume the capabilites of AWS IAM roles.
```

Comment:
```suggestion
For example, EKS clusters leverage this so that they can issue service account tokens which can assume the capabilities of AWS IAM roles.
```